libs/SalesforceSDK/src/com/salesforce/androidsdk/security/SalesforceKeyGenerator.java

```diff
@@ -135,6 +160,49 @@ public static String getSHA256Hash(String privateKey) {
         return hashedString;
     }
 
+    /**
+     * Generate a keypair and get the encoded public key string.
+     *
+     * @param name Alias of the entry in which the generated key will appear in Android KeyStore.
+     * @return RSA public key string.
+     */
+    @Nullable
```

Comment:
Remove `@Nullable` annotation please. We don't use annotations from the `support` library in our SDK.